src/satellite_images_nso/_nso_data_extraction/nso_api.py

```diff
@@ -219,19 +228,30 @@ def check_if_geojson_in_region(row, geojson, max_diff):
     geojson_shape_xy = np.array(geojson_shape_xy)
     xy_intersection = np.array(xy_intersection)
 
+    missing_region = geojson_shape.difference(row_shape)
+
     try:
         print(geojson_shape_xy)
         # This checks if pixels values are the same for both of the shapes.
         # If this isn't the case there should be min or max fluctations based on pixels which is filter with a threshold value.
         difference_array = np.intersect1d(geojson_shape_xy, xy_intersection)
 
+        diff_geojson_sat = (
+            difference_array.shape[0] / geojson_shape_xy.flatten().shape[0]
+        )
+
         # Check if the difference is under a certain percentage.
-        if difference_array.shape[0] / geojson_shape_xy.flatten().shape[0] >= max_diff:
+        if diff_geojson_sat >= max_diff:
             return_statement = True
 
     except Exception as e:
         print(e)
         print(row)
         logging.error(e + " " + row)
 
-    return return_statement
+    return (
```

Comment:
Suggestion: You might want to add the return statement in the docstring for this function.

Reply:
I will update all the documentation in a new pr